packages/lexicon/lightning.d.ts

```diff
@@ -0,0 +1,161 @@
+import type { SanitizePolicies } from '@openfn/logger';
+
+type StepId = string;
+
+/**
+ * Type definitions for Lightning and Worker interfaces
+ *
+ * This is the lightning-worker contract
+ *
+ * It is helpful to have these in the lexicon to avoid a circular dependency between lightning and the worker
+ * It's also kinda nice that the contract isn't in the worker itself, it's on neutral ground
+ */
+// An run object returned by Lightning
+export type Run = {
```

Comment:
im a little surprised that this isn't called `Plan`, but i'm reading on to see what comes next!